guide/contribute/contribute.md

```diff
@@ -102,7 +102,10 @@ Creating visuals for the Bitcoin Design Guide? We recommend following these best
 Layout and formatting options from headers and blockquotes to images and embeds.
 
 ---
+### [Case Studies]({{ '/guide/contribute/case-studies/' | relative_url }})
 
+Tips and tricks on how to share your design or research process in the form of a case study.
+---
```

Comment:
```suggestion

---
```
Make sure to preview this text. It's rendered really big in the site. Not sure exactly why, but it probably just needs an extra line break between the text and the dashes.